inference-engine/src/gna_plugin/transformations/swap_input_matmul_gna.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <transformations_visibility.hpp>
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace ngraph {
+namespace pass {
+
+    class TRANSFORMATIONS_API SwapInputMatMul;
```

Comment:
This definition causes linking errors on Windows during linking of GNAPlugin lib.
I'd remove SwapInputMatMul from ngraph namespace and don't use this definition for it.